R/mincolumns.R

```diff
@@ -131,9 +131,9 @@ extremeCols <- function(x, function.name, remove.missing = TRUE, dims)
     } else
     {
         if (isQTable(x) && getDimensionLength(x) > 2 && statisticsPresentInLastDim(x))
-            y <- apply(x, c(2, getDimensionLength(x)), extremum.fun, na.rm = remove.missing)
+            y <- applyWarnOnce(x, c(2, getDimensionLength(x)), extremum.fun, na.rm = remove.missing)
```

Comment:
Related to above, could consolidate the margin index to simply `2:3` since `getDimensionLength` should never be larger than 3.

Reply:
Done